bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/SUSYCascades/SlepSnuCascade_MNeu220/SlepSnuCascade_MNeu220_TuneCP5_13p6TeV-madgraphMLM_param_card.dat

```diff
@@ -0,0 +1,492 @@
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 1000002  # ul 
+        1 2  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 1000004  # cl 
+        1 2  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 1000006  # t1 
+        1 2  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 2000002  # ur 
+        1 2  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 2000004  # cr 
+        1 2  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 2000006  # t2 
+        1 2  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 1000001  # dl 
+        1 -1  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 1000003  # sl 
+        1 -1  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 1000005  # b1 
+        1 -1  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 2000001  # dr 
+        1 -1  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 2000003  # sr 
+        1 -1  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
+Block QNUMBERS 2000005  # b2 
+        1 -1  # 3 times electric charge
+        2 1  # number of spin states (2S+1)
+        3 3  # colour rep (1: singlet, 3: triplet, 8: octet)
+        4 1  # Particle/Antiparticle distinction (0=own anti)
```

Comment:
are the parameters in this file different from the default value in the model, we always suggest people use "customized card" which includes only the modified parameter

Reply:
I have not modified any of the numbers in the section `QUANTUM NUMBERS OF NEW STATE(S) (NON SM PDG CODE)`. Are you suggesting to remove them if they are standard?

Reply:
Since my understanding is that leaving things as they were here shouldn't be a problem, I haven't touched this part.

Reply:
Hi, my question is that there is parameter card in model MSSM_SLHA2, which parameter in your card is different from the defaults in the model. We always recommend to use 'customized card' to include the modified parameters different from the defaults

Reply:
The only parameters that are different from the default are the neutralino masses. Shall I just leave those in the param_card and delete all the rest?

Reply:
please use a customized card with this modified parameter only, and delete parameter card